tests/thirdpartyemailpassword/test_email_delivery.py

```diff
@@ -1086,3 +1089,41 @@ async def test_create_reset_password_link(
     with raises(GeneralError) as err:
         await create_reset_password_link("invalidTenantId", user_info["id"])
     assert "status code: 400" in str(err.value)
+
+
+@mark.asyncio
+async def test_send_reset_password_email(
```

Comment:
this test is wrong.. you need to do similar to the emailpassword one where you check the actual link being used, and that the query params (tenantId, rid and token are correct)